osu.Game/Screens/SelectV2/BeatmapCarousel.cs

```diff
@@ -31,8 +31,14 @@ public partial class BeatmapCarousel : Carousel<BeatmapInfo>
 
         private readonly LoadingLayer loading;
 
+        private readonly BeatmapCarouselFilterMatching matching;
         private readonly BeatmapCarouselFilterGrouping grouping;
 
+        /// <summary>
+        /// Total number of beatmap difficulties displayed with the filter.
+        /// </summary>
+        public int MatchedBeatmapsCount => matching.BeatmapItemsCount;
```

Comment:
Note that this won't be stable during a sort operation. This may become an issue depending on how it's accessed.
I think it would be a good move to ensure it's stable. I'll make the change.

Reply:
7789f4dbb0c68ee2b2753ae9d1fa201c5ff38e54